src/itemset.js

```diff
@@ -406,14 +406,26 @@ define([
         input.attr("name", name);
         return {
             addAutoComplete: function(sources, changeFunction) {
-                input.autocomplete({
-                    source: sources,
-                    minLength: 0,
-                    change: changeFunction,
-                    close: changeFunction
-                }).focus(function (e) {
-                    // populate the list
-                    $(this).autocomplete('search', $(this).val());
+                input.atwho({
+                    at: "",
+                    data: sources,
+                    maxLen: Infinity,
+                    callbacks: {
+                        filter: function(query, data, searchKey) {
+                            return _.filter(data, function(item) {
+                                return item.name.indexOf(query) !== -1 && item.name !== query;
```

Comment:
`item.name !== query` prevents the single-item menu from appearing on exact match, because I found this annoying while testing, but it's a change from the current behavior. Open to changing it back if anyone else feels strongly.